calico-enterprise/reference/installation/_api.md

```diff
@@ -0,0 +1,5991 @@
+Packages
+- [operator.tigera.io/v1](#operatortigeraiov1)
+
+
+{/* value off */}
```

Comment:
```suggestion
{/* vale off */}
```
@tmjd

Reply:
This syntax is for MDX, which may be thrown when you have this as a plain MD file. Might just rename MDX for that and for general consistency.